nixos/modules/services/security/clamav-unofficial-sigs.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.clamav-unofficial-sigs;
+in {
+  options.services.clamav-unofficial-sigs = with types; {
+    enable = mkEnableOption "Unofficial ClamAV signatures";
+
+    user = mkOption {
```

Comment:
Does it make sense to run this as the same user as `clamav` all the time? Why would anyone ever change this value?

Reply:
You're right, there is no need to change the user, I removed the options.